AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -661,7 +666,7 @@ def _lease_task(self, index, new_eta):
     parameters = {'app': self.app, 'queue': self.name, 'id': task_id}
     result = self.db_access.session.execute(select_task, parameters)[0]
     task_info = {
-      'queueName': self.name,
+      'queueName': LONG_QUEUE_FORM.format(app=self.app, queue=self.name),
```

Comment:
The REST API handler returns a JSON safe version of a task.
In the task context we don't always have the project available. 

I was hoping to avoid adding two new args in https://github.com/AppScale/appscale/blob/master/AppTaskQueue/appscale/taskqueue/task.py#L148 to call it with long_queue_form mode on and to give it an app/project ID.
I guess that's the only way though.